### ISSUE TITLE
Archive not used repos

### ISSUE THREAD

**Issue by syphax-bouazzouni** [agroportal/project-management#161]
In the Agroportal github organisation we have multiple repos that are not used / maintained anymore.
We have the archive the ones that we are sure to use anymore. And deploy / use the ones that might be usesfull for our users as services in [services.agroportal   ](http://services.agroportal.lirmm.fr/)
Repos to check 

- [x] https://github.com/agroportal/omhtrest
- [ ] https://github.com/agroportal/ontology_mapping_harvester (need security update) 
- [x] https://github.com/agroportal/2ndAgroHackathon
- [x] https://github.com/agroportal/1stAgroHackathon
- [x] #206
- [x] https://gite.lirmm.fr/ontoportal/fairness_assessment (transfred to https://github.com/agroportal/fairness) 

Special case for this repo https://github.com/ontoportal-lirmm/sparql-client, Jerome forked it to change a line that may cause bugs and did a pull request for it https://github.com/ncbo/sparql-client/pull/5.  In the mean time the NCBO team did the correction by them self https://github.com/ncbo/sparql-client/commit/a1e6beaf6790df68921cbed9bd1719299b9add3e and let Jerome PL open.
So in summary we should close this repo https://github.com/ontoportal-lirmm/sparql-client, and that's why : 
- Not maintaining our version updated to theirs
- We have no changes,
- Our is outdated (with only one commit with no major changes https://github.com/ncbo/sparql-client/commit/fb4a89b420f8eb6dda5190a126b6c62e32c4c0c9).

- [x] https://github.com/agroportal/documentation/issues/166

**Comment by jonquet** [comment]
After discussion  the repos to archived are (be sure the URLs are still reachable) : 

- [x] https://github.com/agroportal/omhtrest
- [x] https://github.com/agroportal/2ndAgroHackathon
- [x] https://github.com/agroportal/1stAgroHackathon
- [x] https://gite.lirmm.fr/ontoportal/fairness_assessment => delete

**Comment by jonquet** [comment]
I do not archive  https://github.com/agroportal/ontology_mapping_harvester  as this is a project on which I expect a follow up. 
The rest is done. 

@syphax-bouazzouni I am considering "deleting" https://github.com/agroportal/agroportal_web_ui now that we have moved to ontoportal-lirmm
What do you think ? 
(I would suggest having a look to https://github.com/agroportal/agroportal_web_ui/commits/master before)

**Comment by syphax-bouazzouni** [comment]
Yes we can remove https://github.com/agroportal/agroportal_web_ui , all of its commits were transferred to https://github.com/ontoportal-lirmm/bioportal_web_ui.

**Comment by jonquet** [comment]
agroportal_web_ui has been erased.
Other changes on gite.lirmm.fr have been done. 
Closing this issue
